invenio_app_ils/acquisition/schemas/acquisition_purchase_orders/acquisition_purchase_order-v1.0.0.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Schema for Acquisition/Purchase Order",
+  "type": "object",
+  "required": ["$schema", "pid", "name"],
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "pid": {
+      "type": "string",
+      "title": "The persistent identifier of acquisition/purchase order"
+    },
+    "document_pid": {
+      "type": "string",
+      "title": "The persistent identifier of its Document"
+    },
+    "vendor_pid": {
+      "type": "string",
+      "title": "The persistent identifier of its Vendor"
+    },
+    "patron_pid": {
+      "title": "The persistent identifier of the patron who made the request",
+      "type": "string"
+    },
+    "status": {
+      "title": "Status of acquisition/purchase order",
+      "type": "string",
+      "enum": [
+        "PENDING",
```

Comment:
I think it is better to validate this in the code and not in the schema in case someone wants to customize, WDYT?